Networking/Networking/Remote/JetpackConnectionRemote.swift

```diff
@@ -13,6 +13,34 @@ public final class JetpackConnectionRemote: Remote {
         super.init(network: network)
     }
 
+    /// Retrieves the information about Jetpack the plugin for the current site.
+    ///
+    public func retrieveJetpackPluginDetails(completion: @escaping (Result<SitePlugin, Error>) -> Void) {
+        let path = String(format: "%@/%@", Path.plugins, Constants.jetpackPluginName)
```

Comment:
nit: I personally feel this is easier to read, but feel free to ignore
```suggestion
        let path = "\(Path.plugins)/\(Constants.jetpackPluginName)"
```

Reply:
Fixed in 7cb48953d255e92cf52db2da51d3ec8c20aa354a 👍